sdk/storage/azure-storage-blob-batch/CHANGELOG.md

```diff
@@ -1,12 +1,21 @@
 # Release History
 
+## 12.3.0 (2020-01-16)
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob-batch_12.3.0/sdk/storage/azure-storage-blob-batch/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob-batch_12.3.0/sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch)
+
+- Upgraded to version 12.3.0 of Azure Storage Blob.
```

Comment:
@alzimmermsft also fixed an issue with this one. We should note that.

Reply:
Good catch, fixed an issue where download to file may write buffers that were released by Netty.

Reply:
I think that's what I put in 12.2.0

Reply:
@alzimmermsft Isnt that the downloadToFile issue? I think Rick is talking about the batch changes

Reply:
The issue about some batch operations being dropped.

Reply:
I added it to 12.2.0 since I think it was already in when we released last